instruqt/custom-fields/03-updating-the-application/assignment.md

```diff
@@ -0,0 +1,120 @@
+---
+slug: updating-the-application
+id: fpxcohlfy6y3
+type: challenge
+title: Updating the Application
+teaser: Update the application to consume the value set for the Custom License Field
+notes:
+- type: text
+  contents: |-
+    In this next challenge we are going to get our hands dirty with some YAML!
+    We are going to update the application manifests to support turning on/off the Super Duper Feature.
+tabs:
+- title: Dev
+  type: terminal
+  hostname: shell
+  workdir: /home/replicant
+difficulty: basic
+timelimit: 600
+---
+
+## Setting Up Dev Environment ##
+
+Before we update the application we need to make sure we have a dev environment set up. We have provided a dev environment for you to use but you could also run the `replicated` command line from other Linux or Mac machine like your laptop. To use the dev environment we provided for you click on the **Dev** tab.
+
+Make sure to set the `REPLICATED_APP` and `REPLICATED_API_TOKEN` environment variables. This is covered in the Replicated CLI track in more detail.
+
+We also need to download the manifests we want to update. To download the latest version of the applicaiton, we'll employ the [release download](https://docs.replicated.com/reference/replicated-cli-release-download) command.
+
+First, we need to get the latest sequence number. To do this run the following command
+
+```
+replicated release ls
+
+```
+
+You should see results similar to this:
+
+```
+SEQUENCE    CREATED                 EDITED                  ACTIVE_CHANNELS
+10          2022-10-05T15:32:40Z    0001-01-01T00:00:00Z    CustomFields
+9           2022-10-05T13:26:25Z    0001-01-01T00:00:00Z    hellohelm,License100
+8           2022-10-04T20:46:30Z    0001-01-01T00:00:00Z
+7           2022-10-04T20:33:16Z    0001-01-01T00:00:00Z    support-bundle
+6           2022-10-04T20:06:59Z    0001-01-01T00:00:00Z
+5           2022-10-04T20:01:59Z    0001-01-01T00:00:00Z
+4           2022-10-04T18:57:23Z    0001-01-01T00:00:00Z
+3           2022-10-03T18:32:48Z    0001-01-01T00:00:00Z
+2           2022-09-28T13:54:29Z    0001-01-01T00:00:00Z
+1           2022-09-20T19:53:57Z    0001-01-01T00:00:00Z    hello-world
+```
```

Comment:
This could use an update. In the new setup, there will be most likely only a single release assigned to the Stable channel.